src/setup-beam.js

```diff
@@ -379,6 +379,14 @@ function isStrictVersion() {
 }
 
 function getVersionFromSpec(spec0, versions0) {
+  let latest
+  Object.keys(versions0).forEach((version) => {
+    if (version.match(/main|master|nightly|latest/g) == null) {
```

Comment:
Is this possibly missing `maint`?

Reply:
`maint` is already being captured by `main`. I haven't made it to an array since I refactored it out when I made a more complete algorithm; the only limitation is for the version starting with `a` or `b,` which a couple of early versions for rebar3 do.